cli/clibase/net.go

```diff
@@ -0,0 +1,50 @@
+package clibase
+
+import (
+	"net"
+	"strconv"
+
+	"github.com/pion/udp"
```

Comment:
This package is no longer maintained (repo archived). We should consider switching to `github.com/pion/transport/v2/udp` if we still need the library.